style.css

```diff
@@ -0,0 +1,535 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: gray;
+  margin: auto;
+}
+
+a {
+  text-decoration: none;
+}
+
+li {
+  list-style-type: none;
+}
+
+.flex {
+  display: flex;
+}
```

Comment:
- Great job implementing the desktop version of your portfolio. The website is responsive only on screen size 768px, 1024px and 1440px, but seems to be broken on mobile screen(375px). I think this could be because your css currently lacks styling for mobile screen. Can we fix this? 🤔 

Here is what you have displayed on mobile 👇🏽 

| Your portfolio on mobilescreen  |
| :---:   | 
|<img width="392" alt="Screenshot 2023-04-08 at 00 08 04" src="https://user-images.githubusercontent.com/99246359/230704554-07252595-bb5a-4b9e-bb58-51634f384869.png">  |

| Your portfolio on mobilescreen |
| :---:   | 
|<img width="402" alt="Screenshot 2023-04-08 at 00 08 21" src="https://user-images.githubusercontent.com/99246359/230704583-69f36a7d-a37c-40a0-96ee-812ceb085887.png">  |

Reply:
<img width="796" alt="image" src="https://user-images.githubusercontent.com/77242944/230705021-1e83de3c-2690-40e4-af51-ecfedb874f36.png">

Greetings, I will change the background color based on Figma design.
But I think the project requirement for this session was only about desktop version. As it was in need , I focused only on Desktop version style.

Reply:
Hi @mhnazary, kindly note that It is very important to have both the mobile and desktop versions working perfectly, this is all the meaning of responsiveness.